.github/workflows/test.yml

```diff
@@ -22,13 +22,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # the current production setup
           - java: '8'
-            mongo: 'mongodb-linux-x86_64-3.6.13'
-            gradle_test: 'test'
+            mongo: 'mongodb-linux-x86_64-ubuntu2204-7.0.4'
+            wired_tiger: 'false'
           - java: '11'
             mongo: 'mongodb-linux-x86_64-3.6.23'
```

Comment:
Should change this to the actual version that's running in Spin

Reply:
If the version in spin is 4.2+ there's no need for the wired tiger matrix entry at all. If it's lower than that, then the test config setup needs to be modified as well

Reply:
Based on Jason's response, the image is mongo:4, which I assume is lower than version 4.2 and therefore requires switching to mongodb-linux-x86_64-4.0?

Reply:
👍

Reply:
To be clear, you need to get the mongo server version from Jason. It *might* be 4.0, we don't know based soley on the tag

Reply:
done